### PR TITLE
Adding a repository for an Epson G364 IMU driver

### PR DESCRIPTION
**Pull request by jfkeller**
I would like to add a ROS package that is a driver for the Epson G364 IMU.

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -2886,6 +2886,21 @@ repositories:
       url: https://github.com/RIVeR-Lab/epos_hardware.git
       version: indigo-devel
     status: maintained
+  epson_g364_imu_driver:
+    doc:
+      type: git
+      url: git@bitbucket.org:castacks/epson_g364_imu_driver.git
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: You must use https urls otherwise anonymous users can't fetch the ssh urls.

File: indigo/distribution.yaml [hunk 2]
```diff
@@ -2886,6 +2886,21 @@ repositories:
       url: https://github.com/RIVeR-Lab/epos_hardware.git
       version: indigo-devel
     status: maintained
+  epson_g364_imu_driver:
+    doc:
+      type: git
+      url: https://kellerj2@bitbucket.org/castacks/epson_g364_imu_driver.git
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: You need to remove your user name from the URL.

**Comment by tfoote** [comment]
You are indicating that you want it to document and index the `indigo-devel` branch however your repo only has a [master branch](https://bitbucket.org/castacks/epson_g364_imu_driver/branches/) 

And the release section should be a release repository, it should not be the same as the source repository. Typically this is generated by bloom, otherwise you should leave it out. http://wiki.ros.org/bloom

**Comment by dirk-thomas** [comment]
Closing due to no response.

**Pull request closed without merging**